packages/salesforcedx-vscode-core/src/commands/forceLightningAppCreate.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import {
+  CliCommandExecutor,
+  Command,
+  SfdxCommandBuilder
+} from '@salesforce/salesforcedx-utils-vscode/out/src/cli';
+import * as path from 'path';
+import { Observable } from 'rxjs/Observable';
+import * as vscode from 'vscode';
+import { channelService } from '../channels';
+import { nls } from '../messages';
+import { notificationService } from '../notifications';
+import { CancellableStatusBar, taskViewService } from '../statuses';
+import {
+  CompositeParametersGatherer,
+  ContinueResponse,
+  DirFileNameSelection,
+  SelectDirPath,
+  SelectFileName,
+  SfdxCommandlet,
+  SfdxCommandletExecutor,
+  SfdxWorkspaceChecker
+} from './commands';
+
+const LIGHTNING_APP_EXTENSION = '.app';
+
+class ForceLightningAppCreateExecutor extends SfdxCommandletExecutor<
+  DirFileNameSelection
+> {
+  public build(data: DirFileNameSelection): Command {
+    return new SfdxCommandBuilder()
+      .withDescription(nls.localize('force_lightning_app_create_text'))
+      .withArg('force:lightning:app:create')
+      .withFlag('--appname', data.fileName)
+      .withFlag('--outputdir', data.outputdir)
+      .build();
+  }
+
+  public execute(response: ContinueResponse<DirFileNameSelection>): void {
+    const cancellationTokenSource = new vscode.CancellationTokenSource();
+    const cancellationToken = cancellationTokenSource.token;
+
+    const execution = new CliCommandExecutor(this.build(response.data), {
+      cwd: vscode.workspace.rootPath
+    }).execute(cancellationToken);
+
+    execution.processExitSubject.subscribe(async data => {
+      if (
+        data != undefined &&
+        data.toString() === '0' &&
+        vscode.workspace.rootPath
+      ) {
+        vscode.workspace
+          .openTextDocument(
+            path.join(
```

Comment:
The lightning:app:create command will create 6 files. I asked @kaplanjosh whether we should open just the markup or all 6 files being created and concluded that the file that represents the app should be opened (in this case, the .app file).